index.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const { BookendInterface } = require('screwdriver-build-bookend');
+const STORE_URL = 'https://store.screwdriver.cd/v1/builds';
```

Comment:
I don't think we should hardcode this. Maybe this bookend should take in a config with the store-url